scene/animation/animation_tree.cpp

```diff
@@ -750,11 +776,13 @@ void AnimationTree::_animation_node_renamed(const ObjectID &p_oid, const String
 }
 
 void AnimationTree::_animation_node_removed(const ObjectID &p_oid, const StringName &p_node) {
-	ERR_FAIL_COND(!property_reference_map.has(p_oid));
-	String base_path = String(property_reference_map[p_oid]) + String(p_node);
-	for (const PropertyInfo &E : properties) {
-		if (E.name.begins_with(base_path)) {
-			property_map.erase(E.name);
+	for (auto &parent_path : instance_paths[p_oid]) {
```

Comment:
`auto` is [not allowed](https://contributing.godotengine.org/en/latest/engine/guidelines/cpp_usage_guidelines.html#auto-keyword)